packages/communication-react/package.json

```diff
@@ -40,7 +40,8 @@
     "react-linkify": "^1.0.0-alpha",
     "reselect": "~4.0.0",
     "uuid": "^8.1.0",
-    "@fluentui/react-hooks": "~8.3.8"
+    "@fluentui/react-hooks": "~8.3.8",
+    "@fluentui/react-file-type-icons": "~8.5.8"
```

Comment:
sort alphabetically

Reply:
Are we doing that every time we add a dependency?
Seems tedious :D
What's the utility for sorting these?

Reply:
You can do: vs code -> highlight lines -> ctrl + shift + p -> Sort Ascending

Reply:
I know I know ahhaha. I always forget tho. :P

Reply:
I wanted to understand the benefit of sorting these every time. :)

Reply:
Within reason, keep all lists sorted alphabetically. Avoids discussions on the "right order", and makes it trivial for tooling to enforce / auto-fix the order.

As an example, the import statements added via tooling are always sorted.

Reply:
Ooo 'the utility' not 'a utility' sorry misread that lol. It just becomes a mess otherwise. If someone wants to check what dependencies are being used by a package/packlet - it's awful to look through when they are not sorted - it also means to find a package to update it you _have_ to do ctrl+f as finding manually in an unsorted list is impossible

This is definitely the sort of thing we should have prettier do though

Reply:
Let me see if I can make prettier do it automatically ;)